e2e/smoke-tests-suite/panels_smokescreen.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { e2e } from '@grafana/e2e';
+
+// TODO: Better way to generate this list (ideally including all panels >=beta state) (list is case sensitive)
+const PANELS_TO_CHECK = [
+  'Time series',
+  'Bar chart',
+  'Stat',
+  'Gauge',
+  'Bar gauge',
+  'Table',
+  'Pie chart',
+  'State timeline',
+  'Heatmap',
+  'Status history',
+  'Histogram',
+  'Text',
+  'Alert list',
+  'Dashboard list',
+  'News',
+  'Annotations list',
+  'Candlestick',
+  'Canvas',
+  'Flame Graph',
+  'Geomap',
+  'Logs',
+  'Node Graph',
+  'Traces',
+  'XY Chart',
+];
+
+e2e.scenario({
+  describeName: 'Panels smokescreen',
+  itName: 'Tests each panel type in the panel edit view to ensure no crash',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.addDashboard();
+
+    // TODO: Try and use e2e.flows.addPanel() instead of block below
+    try {
+      e2e.components.PageToolbar.itemButton('Add panel button').should('be.visible');
+      e2e.components.PageToolbar.itemButton('Add panel button').click();
+    } catch (e) {
+      // Depending on the screen size, the "Add panel" button might be hidden
+      e2e.components.PageToolbar.item('Show more items').click();
+      e2e.components.PageToolbar.item('Add panel button').last().click();
+    }
+    e2e.pages.AddDashboard.itemButton('Add new visualization menu item').should('be.visible');
+    e2e.pages.AddDashboard.itemButton('Add new visualization menu item').click();
+
+    // Loop through every panel type and ensure no crash
+    PANELS_TO_CHECK.forEach((panel) => {
+      e2e.components.PanelEditor.toggleVizPicker().click();
+      e2e.components.PluginVisualization.item(panel).scrollIntoView().should('be.visible').click();
+
+      // Wait for panel to load (TODO: Better way to do this?)
+      cy.wait(500);
+
+      e2e.components.PanelEditor.toggleVizPicker().should((e) => expect(e).to.contain(panel));
+
+      // TODO: Come up with better check / better failure messaging to clearly indicate which panel failed
+      cy.contains('An unexpected error happened').should('not.exist');
```

Comment:
Not super happy with this check, briefly looked into possibility of adding a selector for our error boundary component which may be better / more extensible. Thoughts?